packages/rspack/tests/cases/parsing/asi/index.js

```diff
@@ -0,0 +1,52 @@
+import { a as b, callme, getCount } from "./a";
```

Comment:
This case is identical to `﻿webpack-test/case/parsing/asi`. Therefore, I think you could possibly remove ﻿`webpack-test/case/parsing/asi/test.filter.js` directly.

Reply:
Furthermore, you could consider deleting the case due to its redundancy :D